beamer/tests/agent/unit/test_handlers.py

```diff
@@ -1,131 +1,25 @@
 from copy import deepcopy
-from pathlib import Path
```

Comment:
I don't mind, do you?

My hope is that we eventually don't even need to run ganache for those and they become even quicker.

Reply:
> I don't mind, do you?

Nah, I guess it just feels an extra level for no real benefit, at least right now. But maybe it will grow.

> My hope is that we eventually don't even need to run ganache for those and they become even quicker.

That would be nice, indeed.

Reply:
> Nah, I guess it just feels an extra level for no real benefit, at least right now. But maybe it will grow.

In that case I'd rather keep it and rather split the existing `test_handlers.py` so that it's more clear.